### PR TITLE
Issue templates for bug reports and feature requests

### PR DESCRIPTION
**Pull request by eugene-manuilov**
Addresses #68

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+## Screenshots
+
+<!-- If applicable, please add screenshots to help explain your problem. -->
+
+## Additional Context
+
+<!-- Please complete the following information. -->
+ - PHP Version: 
+ - OS: [e.g. iOS]
+ - Browser [e.g. chrome, safari]
```
Comment (mitogh) [inline .github/ISSUE_TEMPLATE/bug_report.md]:
```suggestion
 - Browser: [e.g. chrome, safari]
```

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+ - Plugin Version [e.g. 22]
```
Comment (mitogh) [inline .github/ISSUE_TEMPLATE/bug_report.md]:
```suggestion
 - Plugin Version: [e.g. 22]
```

**Review by mitogh — CHANGES_REQUESTED** [review]
Missing some characters, other than that looks great.

**Review by mitogh — APPROVED** [review]

**Comment by jeffpaul** [comment]
I would strongly recommend changing these to YML files to use the [issue forms](https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/configuring-issue-templates-for-your-repository#creating-issue-forms) and help force some of the issue content we're wanting/needing.  [Examples here](https://github.com/10up/simple-podcasting/tree/develop/.github/ISSUE_TEMPLATE).

**Comment by felixarntz** [comment]
@jeffpaul IMO we can still merge this as a starting point (better than not having any), but I like your idea - would you mind to open an issue about using YML files instead so we can discuss there and iterate?

**Comment by JustinyAhin** [comment]
> @jeffpaul IMO we can still merge this as a starting point (better than not having any), but I like your idea - would you mind to open an issue about using YML files instead so we can discuss there and iterate?

I can go ahead and create and issue and a draft PR @felixarntz

**Review by felixarntz — APPROVED** [review]

**Comment by felixarntz** [comment]
@JustinyAhin Sounds great - could you follow up on this?

**Pull request merged**